src/wing/CCPACSWingCell.h

```diff
@@ -100,15 +101,31 @@ class CCPACSWingCell : public generated::CPACSWingCell, public ITiglGeometricCom
         EtaXsi outerTrailingEdgePoint;
     };
 
+    // This struct is used to annotate all the faces of the
+    // wing skin. The annotations are used in the lofting algorithm
+    struct TrimShapeAnnotation
+    {
+        double ccmin, ccmax; // chordwise contour coordinate bounds
+        double scmin, scmax; // spanwise contour coordinate bounds
+        bool keep = true;    // a label, specifying if the face
+                             // will be part of the resulting shape
+    };
+
```

Comment:
I am wondering, whether these details should be part of the public API, in particular if you think, that it will be exposed to Python etc. IMHO, these details would be better places inside the *.cpp file of WingCell.

Reply:
As an internal variable I don't think it needs to be exposed to Python. But I am not sure I understand what you mean, since I have to declare the Annotation struct in the header, because I use it in the `GeometryCache`, don't I? In a way, the `GeometryCache` is also an internal variable that shouldn't be exposed in the Python bindings, right?

BTW, I moved the declaration of `TrimShapeAnnotation` into the `GeometryCache` struct as a nested struct, because it is only ever really used there.

Reply:
Great, and never mind. I had the pimpl ideom in my mind, that allows you to hide these things.